src/web/server/htmlTemplate.ts

```diff
@@ -105,38 +107,80 @@ export const htmlTemplate = ({
+    // "use preconnect when you know for sure that you’re going to be accessing a resource"
+    // - https://www.smashingmagazine.com/2019/04/optimization-performance-resource-hints/
+    // DNS-prefetch should be used for other third party domains that we are likely to connect to but not sure (ads)
+    // Preconnecting to too many URLs can reduce page performance
+    // DNS-prefetch can also be used as a fallback for IE11
+    // More information on preconnecting:
     // https://css-tricks.com/using-relpreconnect-to-establish-network-connections-early-and-increase-performance/
+    // More information on prefetching:
+    // https://developer.mozilla.org/en-US/docs/Web/Performance/dns-prefetch
+
+    // Preconnect links that are only necessary for ads:
+    const staticPreconnctAdUrls = isAdFreeUser
+        ? []
+        : [`https://googleads.g.doubleclick.net`];
+
+    // Prefetch links that are only necessary for ads:
+    const staticPrefetchAdUrls = isAdFreeUser
+        ? []
+        : [
+              `https://ad.crwdcntrl.net`,
+              `https://api.permutive.com`,
+              `https://cdn-gl.imrworldwide.com`,
+              `https://cdn.adsafeprotected.com`,
+              `https://cdn.brandmetrics.com`,
+              `https://cdn.permutive.com`,
+              `https://collector.brandmetrics.com`,
+              `https://confiant-integrations.global.ssl.fastly.net`,
+              `https://ib.adnxs.com`,
+              `https://pagead2.googlesyndication.com`,
+              `https://pixel.adsafeprotected.com`,
+              `https://pubads.g.doubleclick.net`,
+              `https://sb.scorecardresearch.com`,
+              `https://secure-gl.imrworldwide.com`,
+              `https://securepubads.g.doubleclick.net`,
+              `https://static.ads-twitter.com`,
+              `https://stats.g.doubleclick.net`,
+              `https://tpc.googlesyndication.com`,
+              `https://www.google.co.uk`,
+              `https://www.google.com`,
+              `https://www.googleadservices.com`,
+          ];
+
     const staticPreconnectUrls = [
         `${CDN}`,
         `https://i.guim.co.uk`,
-        `https://interactive.guim.co.uk`,
-        `https://www.google-analytics.com`,
+        `https://securepubads.g.doubleclick.net`,
+        `https://j.ophan.co.uk`,
+        `https://ophan.theguardian.com`,
+        ...staticPreconnctAdUrls,
     ];
 
-    // Information on prefetching:
-    // https://developer.mozilla.org/en-US/docs/Web/Performance/dns-prefetch
     const staticPrefetchUrls = [
-        `${CDN}`,
-        `https://i.guim.co.uk`,
+        ...staticPreconnectUrls,
+        ...staticPrefetchAdUrls,
         `https://api.nextgen.guardianapps.co.uk`,
         `https://hits-secure.theguardian.com`,
-        `https://j.ophan.co.uk`,
-        `https://ophan.theguardian.com`,
+        `https://interactive.guim.co.uk`,
+        `https://ipv6.guim.co.uk`,
```

Comment:
That fails to resolve in my browser with my connection (no ipv6) , could that be an issue?